src/common/logger.cc

```diff
@@ -70,6 +70,7 @@
 #include <fmt/format.h>
 #include <fmt/ostream.h>
 #include <fmt/printf.h>
+#include <array>
```

Comment:
@erasmomontes this is an unrelated change. Consider submitting this fix in a separate PR.